source/includes/gcp/_routers.md

```diff
@@ -243,5 +243,23 @@ Add one or more BGP peers associated to existing router interface(s)
 | `bgpPeers[*].name`<br/>_string_          | The name of the BGP peer association.                                                        |
 | `bgpPeers[*].interfaceName`<br/>_string_ | The router interface through which this BGP peering will be realized.                        |
 | `bgpPeers[*].ipAddress`<br/>_string_     | The router's IP address on the above interface for this BGP peering. Only IPv4 is supported. |
-| `bgpPeers[*].peerIpAddress`<br/>_string_ | The peer IP address this BGP peering tries to connect to. Only IPv4 is supported.   |
+| `bgpPeers[*].peerIpAddress`<br/>_string_ | The peer IP address this BGP peering tries to connect to. Only IPv4 is supported.            |
```

Comment:
Could we perhaps change this to "The peer IP address that the BGP peering will try to connect to. Only IPv4 is supported."

Reply:
Changed in 3573f35ba6eea283f9bc657b837f7e2601e9be49